project_status/views/project.xml

```diff
@@ -0,0 +1,93 @@
+<odoo>
+  <!-- List View-->
+  <record id="project_status_view_inherit_list" model="ir.ui.view">

```

Comment:
Would be nice if you use the original id (optional)

Reply:
I've changed the id to the original's id. Thanks.